tools/gnali/gnali.xml

```diff
@@ -0,0 +1,77 @@
+<tool id="gnali" name="gNALI" version="0.1.0" python_template_version="3.6">
+    <description>Get nonessential, LoF variants</description>
+    <requirements>
+        <requirement type="package" version="0.1">gnali</requirement>
```

Comment:
@xialiu71 @Takadonet this should be 0.1.0.

Our bot tried to created a container for it but failed.